ports/gz-msgs/portfile.cmake

```
@@ -34,3 +34,8 @@ endif()
 if(VCPKG_LIBRARY_LINKAGE STREQUAL "static")
   file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/bin" "${CURRENT_PACKAGES_DIR}/bin")
 endif()
+
+if(VCPKG_TARGET_IS_WINDOWS)
+   file(GLOB PROTO_DLLS "${CURRENT_HOST_INSTALLED_DIR}/tools/protobuf/*.dll")
+   file(COPY ${PROTO_DLLS} DESTINATION "${CURRENT_PACKAGES_DIR}/tools/${PORT}${VERSION_MAJOR}")
+endif()
```

```suggestion
vcpkg_copy_tool_dependencies("${CURRENT_PACKAGES_DIR}/tools/${PORT}${VERSION_MAJOR}")
```
if the they are linked.

---

done.